tools/node.js

```diff
@@ -23,6 +23,36 @@ new Function("exports", function() {
     return code.join("\n\n");
 }())(exports);
 
+function to_comment(value) {
+    if (typeof value != "string") value = JSON.stringify(value, function(key, value) {
+        return typeof value == "function" ? "<[ " + value + " ]>" : value;
+    }, 2);
+    return "// " + value.replace(/\n/g, "\n// ");
+}
+
+if (process.env["UGLIFY_BUG_REPORT"]) exports.minify = function(files, options) {
+    var code = [
+        "//=============================================================",
+        "// UGLIFY_BUG_REPORT",
+        to_comment(options),
+        "",
+    ];
+    if (typeof files == "string") {
+        code.push("//-------------------------------------------------------------")
+        code.push("// INPUT CODE", files, "");
+    } else for (var name in files) {
+        code.push("//-------------------------------------------------------------")
+        code.push(to_comment(name), files[name], "");
+    }
+    code.push("//=============================================================", "");
```

Comment:
Don't need the trailing comment line. It would interfere with a trailing source map comment in the original input.

Reply:
Removed in https://github.com/mishoo/UglifyJS/pull/4516/commits/3e0727810b26d156a9012bc1b06e89b60161bbc4